### PR TITLE
[xray-exporter] Add test execution timings

### PR DESCRIPTION
**Pull request by uarlouski**

### CODE REVIEW

File: vividus-to-xray-exporter/src/main/java/org/vividus/xray/factory/TestExecutionFactory.java [hunk 1]
```diff
@@ -89,4 +96,11 @@ private static TestExecutionItemStatus calculateStatus(AbstractStepsContainer st
                      .allMatch(step -> "successful".equals(step.getOutcome())) ? TestExecutionItemStatus.PASS
                              : TestExecutionItemStatus.FAIL;
     }
+
+    private static String asOffsetDateTime(long nanos)
+    {
+        long millis = TimeUnit.MICROSECONDS.convert(nanos, TimeUnit.NANOSECONDS);
+        Instant instant = Instant.ofEpochSecond(millis);
+        return OffsetDateTime.ofInstant(instant, ZoneId.systemDefault()).toString();
```
Comment (valfirst) [inline vividus-to-xray-exporter/src/main/java/org/vividus/xray/factory/TestExecutionFactory.java:104]: how is it displayed in XRay? is user's TimeZone displayed somewhere? or is it converted to the server timezone?

Comment (valfirst) [reply vividus-to-xray-exporter/src/main/java/org/vividus/xray/factory/TestExecutionFactory.java:104]: Jira/Xray parses the provided date and displays it using own format

**Review by valfirst — APPROVED** [review]

File: vividus-to-xray-exporter/src/main/java/org/vividus/xray/factory/TestExecutionFactory.java [hunk 2]
```diff
@@ -89,4 +96,11 @@ private static TestExecutionItemStatus calculateStatus(AbstractStepsContainer st
                      .allMatch(step -> "successful".equals(step.getOutcome())) ? TestExecutionItemStatus.PASS
                              : TestExecutionItemStatus.FAIL;
     }
+
+    private static String asOffsetDateTime(long nanos)
+    {
+        long millis = TimeUnit.MICROSECONDS.convert(nanos, TimeUnit.NANOSECONDS);
+        Instant instant = Instant.ofEpochSecond(millis);
```
Comment (valfirst) [inline vividus-to-xray-exporter/src/main/java/org/vividus/xray/factory/TestExecutionFactory.java:103]: something wrong in 2 above lines: microseconds treated as milliseconds, milliseconds treated as seconds

**Review by valfirst — CHANGES_REQUESTED** [review]

File: vividus-to-xray-exporter/src/main/java/org/vividus/xray/factory/TestExecutionFactory.java (hunk 2 above)
Comment (uarlouski) [reply vividus-to-xray-exporter/src/main/java/org/vividus/xray/factory/TestExecutionFactory.java]: confused nanos with millis, fixed

**Review by valfirst — APPROVED** [review]

**Review by ikalinin1 — APPROVED** [review]

**Pull request merged**